From 8c1871d623210e9420da6d2cf20f28a1be92cdad Mon Sep 17 00:00:00 2001
From: Ky-Anh Huynh <kyanh@theslinux.org>
Date: Sat, 3 Aug 2013 21:19:52 +0700
Subject: [PATCH] Feature string will start with @

Using a dash (-) will not make thing clear. We use @
to provide a feature string for any package. Example,

  pacman@some-special-feature
  uim-vi@with-more-clean

TODO: I do not really know if :@ would break something,
but so far I have no problem with that. New package works
---
 pacman/src/pacman-4.1.0/scripts/makepkg.sh.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
index 08dc94d..bc7957d 100644
--- a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
+++ b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
@@ -93,7 +93,7 @@ PACMAN_OPTS=
 
 # Return the feature string to used in filename
 if [[ -n "$PACKAGE_FEATURE" ]]; then
-	_FEATURE_STRING="-$PACKAGE_FEATURE"
+	_FEATURE_STRING="@$PACKAGE_FEATURE"
 else
 	_FEATURE_STRING=""
 fi
-- 
1.8.3

